fixit/common/config.py

```diff
@@ -70,22 +79,18 @@
 def get_validated_settings(
     file_content: Dict[str, Any], current_dir: Path
 ) -> Dict[str, Any]:
+    # Validates the types and presence of the keys
+    config = pkg_resources.read_text(__package__, "config.schema.json")
+    schema = json.loads(config)
+    validate(instance=file_content, schema=schema)
```

Comment:
It's great that the config validation is simpler now.

There is an error on loading the schema file in CI builds: https://github.com/Instagram/Fixit/pull/188/checks?check_run_id=2453268372
Type error: https://github.com/Instagram/Fixit/runs/2453268393